demisto_sdk/commands/common/docker_helper.py

```diff
@@ -84,6 +83,36 @@ def init_global_docker_client(timeout: int = 60, log_prompt: str = ""):
     return DOCKER_CLIENT
 
 
+@functools.lru_cache
+def docker_login(docker_client) -> bool:
+    """Login to docker-hub using environment variables:
+            1. DOCKERHUB_USER - User for docker hub.
+            2. DOCKERHUB_PASSWORD - Password for docker-hub.
+        Used in Circle-CI for pushing into repo devtestdemisto
+
+    Returns:
+        bool: True if logged in successfully.
+    """
+    docker_user = os.getenv("DOCKERHUB_USER")
+    docker_pass = os.getenv("DOCKERHUB_PASSWORD")
+    if docker_user and docker_pass:
+        try:
+            docker_client.login(
+                username=docker_user,
+                password=docker_pass,
+                registry="https://index.docker.io/v1",
+            )
+            ping = docker_client.ping()
+            logger.info(f"dockerhub login {ping=}")
+            return ping
+        except docker.errors.APIError:
+            logger.info("Did not successfully log in to dockerhub")
+            return False
+
+    logger.info("Did not log in to dockerhub")
+    return False
```

Comment:
I see that in case we fail to connect to dockerhub we only return False, how come this does not affect the rest of our flow? Also shouldn't change the info logs to warning or error?